homeassistant/components/weather/__init__.py

```diff
@@ -68,6 +68,7 @@
 ATTR_CONDITION_WINDY_VARIANT = "windy-variant"
 ATTR_FORECAST = "forecast"
 ATTR_FORECAST_CONDITION: Final = "condition"
+ATTR_FORECAST_HUMIDITY: Final = "humidity"
```

Comment:
This change isn't related to this PR and should therefore not be in this PR.

../Frenck

Reply:
aargh... sorry, will review again, thought to have been clean this time